### PR TITLE
Update CLI integration test project structure

### PR DESCRIPTION
**Pull request by filipesilva**
Blocked on https://github.com/angular/angular/pull/32957, needs to go in after so that the size increases due to different default template are validated separately.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: integration/cli-hello-world-ivy-compat/e2e/protractor.conf.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
   specs: [
     './src/**/*.e2e-spec.ts'
   ],
   capabilities: {
-    browserName: 'chrome',
-    chromeOptions: {
-      binary: process.env.CHROME_BIN,
-      args: ['--no-sandbox']
-    }
+    'browserName': 'chrome'
```
Comment (alan-agius4) [inline integration/cli-hello-world-ivy-compat/e2e/protractor.conf.js]: Is this change intended?

Comment (filipesilva) [reply integration/cli-hello-world-ivy-compat/e2e/protractor.conf.js]: Nope, just an oversight, will fix.

**Review by mhevery — APPROVED** [review]

File: integration/_payload-limits.json [hunk 2]
```diff
@@ -2,27 +2,36 @@
   "cli-hello-world": {
     "master": {
       "uncompressed": {
-        "runtime": 1542,
-        "main": 154385,
-        "polyfills": 45541
+        "runtime-es2015": 1485,
+        "runtime-es5": 1485,
```
Comment (IgorMinar) [inline integration/_payload-limits.json]: please remove the es5 sizes, we don't want to break the build when they change.

**Review by IgorMinar — APPROVED** [review]
the rest looks good, thanks!

File: integration/cli-hello-world-ivy-compat/e2e/src/app.e2e-spec.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { AppPage } from './app.po';
+import { browser, logging } from 'protractor';
 
-describe('cli-hello-world-ivy App', () => {
+describe('workspace-project App', () => {
```
[the remark this answers is not in the document]
Comment (filipesilva) [reply integration/cli-hello-world-ivy-compat/e2e/src/app.e2e-spec.ts]: Fixed this one and three others that were also the same.

File: integration/_payload-limits.json (hunk 2 above)
Comment (filipesilva) [reply integration/_payload-limits.json]: Done

File: integration/bazel-schematics/package.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "devDependencies": {
     "@angular/bazel": "file:../../dist/packages-dist/bazel",
-    "@angular/cli": "file:../../node_modules/@angular/cli",
+    "@angular/cli": "8.0.0-beta.15",
```
[the remark this answers is not in the document]
Comment (filipesilva) [reply integration/bazel-schematics/package.json]: That was part of the old rebase and is now gone.

File: integration/_payload-limits.json [hunk 5]
```diff
@@ -35,6 +35,34 @@
       }
     }
   },
+  "cli-hello-world-lazy": {
+    "master": {
+      "uncompressed": {
+        "runtime-es2015": 2289,
+        "runtime-es5": 2286,
```
[the remark this answers is not in the document]
Comment (filipesilva) [reply integration/_payload-limits.json]: Done

File: .circleci/config.yml [hunk 6]
```diff
@@ -746,131 +746,132 @@ workflows:
...
-      - lint:
-          requires:
-            - setup
-      - test:
-          requires:
-            - setup
-      - test_ivy_aot:
-          requires:
-            - setup
+      # - lint:
```
[the remark this answers is not in the document]
Comment (filipesilva) [reply .circleci/config.yml]: It was, yes. While working in these integration PRs I disable the rest of CI to make it faster and less expensive.
